src/mds/MDSRank.cc

```diff
@@ -2658,16 +2658,29 @@ void MDSRankDispatcher::handle_asok_command(
   } else if (command == "session evict" ||
 	     command == "client evict") {
     std::lock_guard l(mds_lock);
+    bool yes_i_really_mean_it = false;
     std::vector<std::string> filter_args;
-    cmd_getval(cmdmap, "filters", filter_args);
+    cmd_getval(cmdmap, "yes_i_really_mean_it", yes_i_really_mean_it);
+    bool got_val = cmd_getval(cmdmap, "filters", filter_args);
```

Comment:
I think it would be better to test `filter_args` vector for emptiness instead of checking `cmd_getval` return value to handle a case where e.g. an empty string `''` is passed.